src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderIdentifier.java

```diff
@@ -62,6 +63,9 @@ public static class OffenderIdentifierPK implements Serializable {
     @Column(name = "CASELOAD_TYPE")
     private String caseloadType;
 
+    @Column(name = "CREATE_DATETIME", insertable = false, updatable = false)
+    private LocalDateTime createDateTime;
```

Comment:
The old SQL solution included the created date for an offender identifier in property `whenCreated` on the API model